site/en/blog/permissions-chip/index.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Permissions request chip'
+description: >
+  Permission prompts demand an answer immediately and are frequently ignored. Chrome 97 adds
+  dedicated space in the location bar for permissions, with the goal of putting all
+  permissions-related UI (prompts, indicators, and controls) in the same place and thereby making
+  the prompts less demanding to reduce the perceived insistence on a decision.
+authors:
+  - thomassteiner
+date: 2022-02-01
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/K7jiiutmZA0DvzGU9irA.jpg
+alt: Computer user in front of a desktop computer.
+tags:
+  - chrome-97
+---
+
+## The permissions UX until now
+
+When a user visits a site that requests a permission, a bubble pops up to prompt the user to make a
+decision. For example, below you can see the geolocation permission prompt as implemented in Chrome
+up to version 96. (You can try this and other permissions on our demo site
+[permission.site](https://permission.site/).)
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/dVZNzWjq3wBsRY4E7ShO.png", alt="Chrome geolocation permission prompt", width="690", height="432" %}
+
+### Most permission prompts are ignored or dismissed
+
+Chrome's telemetry data proves that a lot of permission prompts are ignored. You can explore the
+[notification permission data](https://developers.google.com/web/updates/2020/02/notification-permission-data-in-crux)
+in the Chrome UX report yourself. For now, consider the table below that shows
+how Windows users reacted to the geolocation and the notification prompts on sites in an accumulated way:
+
+<div class="table-wrapper scrollbar">
```

Comment:
Are the geolocation numbers available publicly as part of the CRUX report? Wanted to check it was okay publishing these figures.

Reply:
We just have notifications in there. Let's remove geolocation for now, the point will get across still.